app/services/proofing/lexis_nexis/verification_error_parser.rb

```diff
@@ -47,11 +47,9 @@ def parse_product_error_messages
         return {} if products.nil?
 
         products.each_with_object({}) do |product, error_messages|
-          if product['ProductType'] == 'InstantVerify'
-            next if product['ProductStatus'] == 'pass'
-          elsif product['ProductStatus'] == 'pass'
-            next
-          end
+          next if product['ProductStatus'] == 'pass'
+
+          product.delete('ParameterDetails') # don't log reflected PII
```

Comment:
So, I think we may want the parameter details of some of the products. I think we want to delete parameter details for just the phonefinder product

Reply:
Thank you for the clarity. Changed!